app/views/shared/_footer_revised.html.haml

```diff
@@ -2,31 +2,44 @@
   .primary-footer-nav
     .container
       .row
-        %nav.col-md-3.pull-right
-          %h4
-            = active_link "Donate to Bike Index", support_the_index_path
+        %nav.col-md-2.pull-right
+          %h4 Bike Index
           %ul
-            %li
-              = active_link "Store", shop_path
-            %li
-              = active_link "Ambassadors", ambassadors_how_to_path
-        %nav.col-md-3.pull-right
-          %h4
-            = active_link "Help", help_path
+            %li= active_link "FAQ", support_path
+            %li= active_link "Blog", news_index_path
+            %li= active_link "Help", help_path
+            %li= active_link "About", about_path
+            %li= active_link "Where We Are", where_path
+            %li= active_link "Stolen Bike Recoveries", recovery_stories_path
+
+        %nav.col-md-2.pull-right
+          %h4 Support Us
           %ul
-            %li
-              = active_link "Sign up a shop or organization", new_organization_path
-            %li
-              = active_link "BikeHub", "https://bikehub.com"
-        %nav.col-md-3.pull-right
-          %h4
-            = active_link "Dev and Design Resources", dev_and_design_path
+            %li= active_link "Donate", support_bike_index_path
+            %li= active_link "Store", store_path
+            %li= active_link "Ambassadors", ambassadors_current_path
```

Comment:
I think this should link to the ambassadors how to for now, at least until we update the ambassadors page. I think the title "Ambassadors" is still fine